Dockerfile.Ubuntu1604

```diff
@@ -14,38 +14,44 @@ RUN apt-get -qq update && apt-get -qq install \
     python-numpy \
     python-dev \
     wget \
-    bzip2
+    bzip2 && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Why not do a normal `apt-get clean`?

Reply:
Ubuntu Docker images run `apt-get clean` after each install. [More here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers)